apps/bare-expo/ios/BareExpo/AppDelegate.mm

```diff
@@ -36,6 +36,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   return [super application:application didFinishLaunchingWithOptions:launchOptions];
 }
 
+- (BOOL)bridgelessEnabled {
+  return false;
+}
+
```

Comment:
We should probably remove this, right?

```suggestion 
```

Reply:
Yep, PR is old now, this was needed for one of the RCs